packages/js/src/index.ts

```diff
@@ -2,3 +2,4 @@ export { AxiomWithoutBatching, Axiom, ContentType, ContentEncoding, IngestOption
 export { ClientOptions } from './httpClient.js';
 export { datasets } from './datasets.js';
 export { users } from './users.js';
+export {Logger}  from './logger';
```

Comment:
Would be helpful to export the other types as well, like LoggerOptions and log levels, etc.